src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs

```diff
@@ -21,7 +21,7 @@ public PerformanceTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [ActiveIssue("PerformanceTest")]
```

Comment:
```[Benchmark]``` tests are skipped by default in CI. Build scripts pass ```-notrait Benchmark=true```. So these can only be run on demand by setting xunit Benchmark trait.